src/components/OptionsList/BaseOptionsList.js

```diff
@@ -235,12 +137,8 @@ class OptionsList extends Component {
                     ref={this.props.innerRef}
                     indicatorStyle="white"
                     keyboardShouldPersistTaps="always"
-
-                    // Both `keyboardDismissMode` & `onScrollBeginDrag` props are needed to ensure that virtual keyboard is
-                    // dismissed on all platforms.
-                    // eslint-disable-next-line react/jsx-props-no-multi-spaces
-                    keyboardDismissMode="on-drag"
-                    onScrollBeginDrag={() => Keyboard.dismiss()}
+                    keyboardDismissMode={this.props.keyboardDismissMode}
+                    onScrollBeginDrag={this.props.onScrollBeginDrag}
                     contentContainerStyle={[...this.props.contentContainerStyles]}
```

Comment:
Could you please make it so
```suggestion
                    contentContainerStyle={this.props.contentContainerStyles}
```